spec/models/location_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+describe Location do
+  context "validations" do
+    it { is_expected.to validate_presence_of(:creator)}
```

Comment:
Styling thing to keep in mind: add a space on the closing curlies as well. E.g., `{ is_expected.to test }`